src/articles/ExtensionMessage.js

```diff
@@ -1,18 +1,22 @@
 import Modal from "../components/modal_shared/Modal";
-import Header from "../components/modal_shared/Header";
-import Heading from "../components/modal_shared/Heading";
-import Main from "../components/modal_shared/Main";
+import { Header } from "../components/modal_shared/Header.sc";
+import { Heading } from "../components/modal_shared/Heading.sc";
+import { Main } from "../components/modal_shared/Main.sc";
 import FullWidthImage from "../components/FullWidthImage";
-import Footer from "../pages/_pages_shared/Footer";
-import ButtonContainer from "../components/modal_shared/ButtonContainer";
-import GoToButton from "../components/modal_shared/GoToButton";
+import { Footer } from "../components/modal_shared/Footer.sc";
+import { ButtonContainer } from "../components/modal_shared/ButtonContainer.sc";
+import { Button } from "../pages/_pages_shared/Button.sc";
 import { getExtensionInstallationLinks } from "../utils/extension/extensionInstallationLinks";
 import { isSupportedBrowser } from "../utils/misc/browserDetection";
-import { isMobile } from "../utils/misc/browserDetection";
-import FileDownloadOutlinedIcon from "@mui/icons-material/FileDownloadOutlined";
+import {
+  isMobile,
+  runningInChromeDesktop,
+} from "../utils/misc/browserDetection";
+import ArrowForwardRoundedIcon from "@mui/icons-material/ArrowForwardRounded";
```

Comment:
I agree. However, this name is defined by the creators of MUI icons, and if I change it, the import link will not match.

Reply:
ah, they should read https://writings.stephenwolfram.com/2010/10/the-poetry-of-function-naming/ by Wolfram :)

And in this case, we're lucky. Because it's a default import you can name it whatever you like ;)

Reply:
so `import RoundedForwardArrow from "@mui/icons-material/ArrowForwardRounded";` should work like a breeze.

Reply:
✅ Done